exporters/prometheus/include/opentelemetry/exporters/prometheus/prometheus_exporter_utils.h

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "opentelemetry/sdk/metrics/record.h"
+#include "prometheus/metric_family.h"
+
+namespace prometheus_client = ::prometheus;
```

Comment:
Is there a way to limit the scope of these? I'm worried about having them in a header file.

Reply:
removed it from header, and put it in implementation files